src/OpenSpaceToolkit/Astrodynamics/Flight/System/SatelliteSystem.cpp

```diff
@@ -144,6 +144,22 @@ SatelliteSystem SatelliteSystem::Undefined()
     };
 }
 
+SatelliteSystem SatelliteSystem::Default()
+{
+    const Composite satelliteGeometry(Cuboid(
+        {0.0, 0.0, 0.0}, {Vector3d {1.0, 0.0, 0.0}, Vector3d {0.0, 1.0, 0.0}, Vector3d {0.0, 0.0, 1.0}}, {1.0, 2.0, 3.0}
+    ));
+
+    return {
+        Mass(1000.0, Mass::Unit::Kilogram),
```

Comment:
nit:
```suggestion
        Mass(100.0, Mass::Unit::Kilogram),
```

Might be closer to the average satellite.